app/src/main/java/ch/epfl/favo/chat/ChatPage.java

```diff
@@ -193,7 +204,7 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
   @Override
   public void onStart() {
     super.onStart();
-    attachRecyclerViewAdapter();
+//    attachRecyclerViewAdapter();
```

Comment:
Have you checked if there are any performance issues if the favor is remotely updated while looking at the chat? Im afraid that calling attachRecyclerViewAdapter would cause the list of messages to be recreated when the favor status changes for example.

Reply:
If it's an issue we might have to remove the observer and just pass the whole favor in the bundle as a parcelable object

Reply:
Regarding chat button blocking the title: I changed the font size of the title from 20sp to 16sp so hopefully it can fit 30 characters.